### PR TITLE
Remove audioDeviceInformation state bug fix

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**
Remove audioDeviceInformation, since it is unused and fixes a bug.

**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? Run integration tests, unit tests.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? No
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/audiovideocontroller/AudioVideoControllerState.ts [hunk 1]
```diff
@@ -110,8 +110,6 @@ export default class AudioVideoControllerState {
 
   videoInputAttachedTimestampMs: number = 0;
 
-  audioDeviceInformation: { [id: string]: string } = {};
```
Comment (XHatan) [inline src/audiovideocontroller/AudioVideoControllerState.ts:113]: we do publish video device information through event controller.  I am more curious about how to trigger the track label undefined condition ? Any idea ?

Comment (michhyun1) [reply src/audiovideocontroller/AudioVideoControllerState.ts:113]: This was due to a bug filed here: https://github.com/aws/amazon-chime-sdk-js/issues/1103. Looks like the device label is undefined

**Review by devalevenkatesh — APPROVED** [review]

**Review by XHatan — APPROVED** [review]

File: guides/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
```
Comment (ltrung) [inline guides/index.ts:1]: Is this intentional? Doesn't seem related.
